main.py

```diff
@@ -12,29 +13,19 @@ def create_tables():
 
 @app.command()
 def add_expense(
-    expense_info: str = typer.Option(..., help="Enter the expense details in the format: Date,Category,Amount,Description"),
+    Date: str = dt.datetime.now(),
```

Comment:
## GOOD PRACTICE
import datetime as dt
now = dt.datetime(2023, 2, 2, 2, 2, 2)

Your recommendation was this but I took the liberty to just make it so that it takes the current time from the system instead of putting it manually  ( not sure if that was meant from the beginning but the extra numbers confused me 😅)

Reply:
Yeah `dt.datetime.now()` is what you do to get the time now. Ok for me. 

Just make sure that a variable is snake_case and and only classes themselved are camelcase.

When using `datetime`, your variable names should be called whatever they provide the time for.

For example:

```python
created_at = dt.datetime.now()  # 20/10/2023 8:20
created_data = dt.date.today() # 20/10/2023
```